src/styles/customizations/DegenApeAcademy.css

```diff
@@ -411,6 +417,21 @@
       color: white;
       background: #1d1c1c;
       font-family: 'Raleway', 'sans-serif';
```

Comment:
fonts are still broken on this page, not sure if this has anything to do with it

Reply:
I removed that, and updated the preview link